### PR TITLE
feat(capacity): add limits for number of CUs per CC, add max CC duration limit

### PR DESCRIPTION
**Pull request by ValeryAntopol**

### CODE REVIEW

File: integration-tests-contracts/src/commitment.test.ts [hunk 1]
```diff
@@ -198,7 +198,7 @@ describe.sequential("Capacity commitment", () => {
 
     assert(commitmentId, "Commitment ID doesn't exist");
 
-    await DepositCC([commitmentId], registeredOffer.peers, CC_DURATION_DEFAULT);
+      await DepositCC([commitmentId], registeredOffer.peers, CC_DURATION_DEFAULT);
```
Comment (crystalbit) [inline integration-tests-contracts/src/commitment.test.ts:201]:
```suggestion
    await DepositCC([commitmentId], registeredOffer.peers, CC_DURATION_DEFAULT);
```

File: integration-tests-contracts/src/commitment.test.ts [hunk 2]
```diff
@@ -240,6 +240,124 @@ describe.sequential("Capacity commitment", () => {
...
+    let maxCUPerCommitment = await coreContract.maxCUPerCommitment();
+    {
+      await expect(coreContract.setCapacityConstant(
+        CapacityConstantType.MinDuration,
+        0
+      )).rejects.toThrow();
+
+      await expect(coreContract.setCapacityConstant(
+        CapacityConstantType.MinDuration,
+        maxDuration + 1n
```
Comment (crystalbit) [inline integration-tests-contracts/src/commitment.test.ts:258]: could you pls add trailing commas? not sure but there may be linter rule for this

File: integration-tests-clients/src/matcher/dealMatcherClient.test.ts [hunk 3]
```diff
@@ -25,7 +25,7 @@ import { TEST_NETWORK, PROVIDER, WAIT_CONFIRMATIONS } from "../env.js";
 // Empirically measured time for subgraph indexing on 4CPU, 8Gb mem.
 const DEFAULT_SUBGRAPH_TIME_INDEXING = 300000;
 const TESTS_TIMEOUT = 120000 + 30000 + DEFAULT_SUBGRAPH_TIME_INDEXING;
-const CAPACITY_DEFAULT_DURATION = 60 * 60 * 24 * 30; // 30 days in seconds
+const CAPACITY_DEFAULT_DURATION = 300; // it was 30 * 24 * 60 * 60 epochs previously, lets hope 300 epochs is enough
```
Comment (InversionSpaces) [inline integration-tests-clients/src/matcher/dealMatcherClient.test.ts:28]: nit: Lets add `_EPOCHS` suffix so it is unambiguous what is the unit

**Comment by InversionSpaces** [comment]
utACK

**Review by InversionSpaces — APPROVED** [review]

File: integration-tests-clients/src/matcher/dealMatcherClient.test.ts (hunk 3 above)
Comment (ValeryAntopol) [reply integration-tests-clients/src/matcher/dealMatcherClient.test.ts:28]: oh, great, i will

**Pull request merged**